bld/CLMBuildNamelist.pm

```diff
@@ -2341,6 +2359,17 @@ sub setup_logic_do_transient_pfts {
    if (&value_is_true($nl->get_value($var)) && $cannot_be_true) {
       $log->fatal_error($cannot_be_true);
    }
+
+   # if do_transient_pfts is .true. and any of these (n_dom_* or toosmall_*)
+   # are > 0 or collapse_urban = .true., then give fatal error
+   if (&value_is_true($nl->get_value($var))) {
+      if (&value_is_true($nl->get_value('collapse_urban'))) {
+         $log->fatal_error("$var cannot be combined with collapse_urban");
+      }
+      if ($n_dom_pfts > 0 || $n_dom_landunits > 0 || $toosmall_soil > 0 || $toosmall_crop > 0 || $toosmall_glacier > 0 || $toosmall_lake > 0 || $toosmall_wetland > 0 || $toosmall_urban > 0) {
```

Comment:
Remove this restriction, since we envision using n_dom_* and toosmall_*'s together.

Reply:
@ekluzek , I'm pretty sure this is fine. The logic translates to the following:
- If do_transient_pfts is .true. and ANY of these (n_dom_* or toosmall_*) are > 0  OR collapse_urban = .true., then give fatal error
- Same if do_transient_crops is .true.

Reply:
It's not excluding coexistence of n_dom_* and toosmall_*.
It's excluding coexistence of those with *transient*.

Reply:
Ahh, OK. Sorry I misread that. I thought there was another place where that was done.